### ISSUE TITLE
checksum of tkn_0.34.0_Linux_x86_64.tar.gz is wrong 

### ISSUE THREAD

**Issue by suzuki-shunsuke** [tektoncd/cli#2211]
The checksum of [tkn_0.34.0_Linux_x86_64.tar.gz](https://github.com/tektoncd/cli/releases/download/v0.34.0/tkn_0.34.0_Linux_x86_64.tar.gz) is different from the checksum in [checksums.txt](https://github.com/tektoncd/cli/releases/download/v0.34.0/checksums.txt).

```console
$ sha256sum --ignore-missing -c checksums.txt
tkn_0.34.0_Linux_x86_64.tar.gz: FAILED
sha256sum: WARNING: 1 computed checksum did NOT match
sha256sum: checksums.txt: no file was verified
```

```console
$ sha256sum tkn_0.34.0_Linux_x86_64.tar.gz 
958c86d518a459122e78958fc81932ce44d7109672bfbffa513d11fbec9a91b6  tkn_0.34.0_Linux_x86_64.tar.gz
```

```console
$ grep tkn_0.34.0_Linux_x86_64.tar.gz checksums.txt 
f8404db5b6b846bdf7a81701b352b70279b718d6c690863ef7d3014c7d481a68  tkn_0.34.0_Linux_x86_64.tar.gz
```

checksums.txt

```
888670adcac716294e6824fae7dccb295128937a1c11fadc986334f9b3162414  tektoncd-cli-0.34.0_Linux-64bit.deb
fc351d26d43efa940c59ca8f7645613b038602512cb8e73eaa35e8aab0227b09  tektoncd-cli-0.34.0_Linux-64bit.rpm
285184a8d43d992d1968bcfe2e54ecdb549d1d5b1045b9e7fe275de368edee58  tektoncd-cli-0.34.0_Linux-ARM64.deb
c52ae25f08cb09b0a9a1b956b1ce670f18994c266227df9284e567ff73b0f2a6  tektoncd-cli-0.34.0_Linux-ARM64.rpm
97bf21681cd7109dcb0ef5dd358260001a10d80548a315b9658cde21fb81132f  tektoncd-cli-0.34.0_Linux-ppc64le.deb
e05bdf5648e2bc5d06b7b5df3a6ff0a1fd861b6ac35567925984e8f68dc3efac  tektoncd-cli-0.34.0_Linux-ppc64le.rpm
c895d2723efcfff6e094ce12972bf1e958cfdafb2758d7468d854ea4d48cb6ef  tektoncd-cli-0.34.0_Linux-s390x.deb
5be83c2d1b6b41e7579ba5b19deeb1cea05ff04a798fd536b41aa9c6f54b5074  tektoncd-cli-0.34.0_Linux-s390x.rpm
d77967432c553a8c3487b2b18261b5f1ed802bee928ec77d7aaf60c1801430de  tkn_0.34.0_Darwin_all.tar.gz
3f55dfa6e406958bcca72c2481391177977b9935856da82e88995ddc68d5c406  tkn_0.34.0_Linux_aarch64.tar.gz
3c2462e31c5018ebb8eaac516b5512fa14a33c50638a68deb20461d3f1148cb5  tkn_0.34.0_Linux_ppc64le.tar.gz
15fff434993486b9a59b2c182c0fae1bf4aa04f237913ec2a7dd0b229c4f53a3  tkn_0.34.0_Linux_s390x.tar.gz
f8404db5b6b846bdf7a81701b352b70279b718d6c690863ef7d3014c7d481a68  tkn_0.34.0_Linux_x86_64.tar.gz
162e48e0604edeb0337c8245c79646b9c43413a53933f5800c470cdbb1975df2  tkn_0.34.0_Windows_aarch64.zip
b87ceebeea935522faab40d420d5893d52de01437fb386eeab9556167e6b20fe  tkn_0.34.0_Windows_x86_64.zip
```

**Comment by piyush-garg** [comment]
cc @vinamra28

**Comment by piyush-garg** [comment]
Hey @suzuki-shunsuke 

Thanks for reporting. Release pipeline failed for one time, we triggered it again, and it seems it did not overwrite contents which were already present in the tag.

Is it something blocking or urgent, we can cut a new release or let's see if we get any bug fix to be back ported. Thanks

cc @vdemeester @vinamra28

**Comment by suzuki-shunsuke** [comment]
Thank you for handling the issue.
For me this is not urgent.

Regarding v0.34.0, can we fix the issue by removing all assets and retrying the release workflow?

**Comment by suzuki-shunsuke** [comment]
Oh, I see.

**Comment by piyush-garg** [comment]
we have cut 0.35 release,  hope this will be fixed now for 0.35

**Comment by suzuki-shunsuke** [comment]
Thank you!
It seems the issue was fixed.
